skbio/tree/tests/test_tree.py

```diff
@@ -817,6 +817,15 @@ def test_levelorder(self):
         obs = [n.name for n in self.simple_t.levelorder()]
         self.assertEqual(obs, exp)
 
+    def test_bifurcate(self):
```

Comment:
Can you add a test for non-default `insert_length`? I thought I just saw tests for that...